packages/react-components/src/components/SendBox.tsx

```diff
@@ -243,18 +214,22 @@ export const SendBox = (props: SendBoxProps): JSX.Element => {
     }
 
     // Don't send message until all files have been uploaded successfully
+    /* @conditional-compile-remove(file-sharing) */
     setFileUploadsPendingError(undefined);
 
-    if (hasIncompleteFileUploads(props)) {
-      /* @conditional-compile-remove(file-sharing) */
+    /* @conditional-compile-remove(file-sharing) */
+    if (hasIncompleteFileUploads(activeFileUploads)) {
```

Comment:
Did we made the change?

Reply:
cc @palatter

Reply:
Yes, it was part of the latest Patrick's PR

Reply:
https://github.com/Azure/communication-ui-library/pull/4034

Reply:
Yes was part of my refactor on sendbox.